runtime/jcl/common/java_lang_StackWalker.cpp

```diff
@@ -279,6 +314,39 @@ Java_java_lang_StackWalker_getImpl(JNIEnv *env, jobject clazz, jlong walkStateP)
 				J9VMJAVALANGSTACKWALKERSTACKFRAMEIMPL_SET_CALLERSENSITIVE(vmThread, PEEK_OBJECT_IN_SPECIAL_FRAME(vmThread, 0), TRUE);
 			}
 
+#if JAVA_SPEC_VERSION >= 21
+			if (J9_ARE_ALL_BITS_SET((UDATA)walkState->userData1, GET_MONITORS)) {
+				J9ObjectMonitorInfo *monitorInfo = (J9ObjectMonitorInfo *)walkState->userData3;
+				IDATA *monitorCount = (IDATA *)(&walkState->userData4);
+
+				/* Temp fields to find the number of monitors hold by this frame. */
+				J9ObjectMonitorInfo *tempInfo = monitorInfo;
+				U_32 count = 0;
+				/* Use a while loop as there may be more than one lock taken in a stack frame. */
+				while ((0 != *monitorCount) && ((UDATA)tempInfo->depth == walkState->framesWalked)) {
+					count++;
+					tempInfo++;
+					(*monitorCount)--;
```

Comment:
Using `+= 1` or `-= 1` is preferred over using `++` or `--`:
```c
					count += 1;
					tempInfo += 1;
					*monitorCount -= 1;
```
See https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#operators.

Reply:
fixed